### ISSUE TITLE
table.fetch_all_rows() returns None. 

### ISSUE THREAD

**Issue by vaghawan** [barseghyanartur/starbase#25]
Hi, I'm working with the hbase 1.2.6, while fetch works great, but when I do table.fetch_all_rows() it returns None. I don't have huge dataset, as I'm just trying to work with the small data first. I was just wondering why it was not working. Could you please help? 

Thanks

**Comment by zerotocode398** [comment]
Have you solved the same problem？

**Comment by vaghawan** [comment]
I don't think so, I must have moved on with something else. It's been a long time, I don't remember solving it.

**Comment by barseghyanartur** [comment]
This repository is deprecated.

**Comment by zerotocode398** [comment]
Now I need to use Python to fuzzy query the rowkey of HBase. Is there any good way，ths
